testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -423,6 +424,33 @@ public void testTrinoShowingSparkCreatedTables()
         onTrino().executeQuery("DROP TABLE " + trinoTableName(trinoTable));
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testPrestoWritingDataWithObjectStorageLocationProvider()
```

Comment:
Presto -> Trino

Reply:
We should run this test for all supported formats, since page sink can have format-dependent behavior (currently it does not have). Requires https://github.com/trinodb/trino/pull/8751